.github/workflows/cleanup_issues_before_2021.yml

```diff
@@ -0,0 +1,60 @@
+name: Close Pre-2021 Issues
+on:
+  # schedule:
+    # - cron: '35 15 * * *'  # Runs daily at 3:30 PM - stop after first run
+  workflow_dispatch: # Allows manual triggering from the Github UI
+
+permissions:
+  issues: write  # Required to close issues and post comments
+  contents: read  # Needed by most GitHub Actions
+  pull-requests: read  # Needed to access PR data
+
+jobs:
+  close-inactive:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          
+      - name: Label old issues
+        run: |
+          # Set the cutoff date (issues created before this date will be labeled)
```

Comment:
I'm wondering if we could save a lot of bash logic here by instead using some prebuilt action runners such as the one we use for other labeling automation: https://github.com/strapi/strapi/blob/2a9b9b61cfdc484a1736b8c7d460c850ecc64608/.github/workflows/issues_handleLabel.yml#L23

We already use issue helper: https://github.com/actions-cool/issues-helper
For example: https://actions-cool.github.io/issues-helper/advanced#close-issues

---

Entirely up to you on this, doing it manually in bash with curl and bash logic is totally fine but might be slightly harder to maintain if people don't know bash